core/src/main/scala/org/apache/spark/status/api/v1/api.scala

```diff
@@ -31,6 +33,9 @@ class ApplicationInfo private[spark](
     val memoryPerExecutorMB: Option[Int],
     val attempts: Seq[ApplicationAttemptInfo])
 
+@JsonIgnoreProperties(
+  value = Array("startTimeEpoch", "endTimeEpoch", "lastUpdatedEpoch"),
```

Comment:
Will this exclude the Epoch values from the api? Because if I remember correctly we added those for the api specifically